robottelo/datafactory.py

```diff
@@ -108,6 +109,42 @@ def invalid_emails_list():
     ]
 
 
+@filtered_datapoint
+def invalid_boolean_strings():
+    """Create a list of invalid booleans. E.g not true nor false"""
+
+    def not_boolean_str(s):
+        return s not in ('true', 'false')
+
+    return [
+        gen_alpha(validator=not_boolean_str, default='notboolean')
+        for _ in range(10)
```

Comment:
perhaps add the number 10 as a default argument

Reply:
Already fixed